src/main/scala/com/codacy/client/bitbucket/v2/service/HookServices.scala

```diff
@@ -32,18 +34,22 @@ class HookServices(client: BitbucketClient) {
       hookUrl: String,
       events: Set[String]
   ): RequestResponse[v2.Webhook] = {
-    val servicesUrl = getServicesUrl(author, repo)
+    val servicesUrl = generateHooksUrl(author, repo)
+    val encodedUuid = URLEncoder.encode(uuid, "UTF-8")
     val payload = Json.obj("active" -> active, "description" -> description, "url" -> hookUrl, "events" -> events)
-    client.putJson(Request(s"$servicesUrl/$uuid", classOf[v2.Webhook]), payload)
+    client.putJson(Request(s"$servicesUrl/$encodedUuid", classOf[v2.Webhook]), payload)
```

Comment:
Codacy found an issue: [The string literal '${ .. }/${ .. }' appears 2 times in the file.](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)